src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -554,6 +554,8 @@ private void OutputResultsHelper(IEnumerable<CommandInfo> results)
 
             _timer.Stop();
```

Comment:
Since the tracked elapsed time is not used at all, shall we remove the `_timer` and the related time tracking code?

Reply:
fixed